app/assets/stylesheets/content/_calendars.sass

```diff
@@ -21,6 +21,11 @@ full-calendar
       .fc-button-active
         @extend .button.-active
 
+  .fc-title
+    max-width: 100%
+    overflow: hidden
+    text-overflow: ellipsis
```

Comment:
I explicitly did not want the behaviour of the `text-shortener` which also brings in `word-wrap: nowrap` so that the title is cut off after the first line. As I wanted more lines, I then went with the fade out approach.

Reply:
Yeah, that's a common problem. In such cases, I overwrite the `word-wrap` after the mixin. However I admit, both options are not optimal.

Reply:
I have added a variable to the `text-shortener` and applied it throughout the application where appropriate.